tests/core/test_domain.py

```diff
@@ -1,8 +1,9 @@
 import json
 import pytest
+from _pytest.tmpdir import TempdirFactory
```

Comment:
it would be great if you could add a test that does the whole roundtrip - from loading in a file to the merged data and domain. e.g. creating the files in `data/test_...` and then using that dir as the skills root

Reply:
or directly create an example in `examples`